src/makeInfinityAdder.js

```diff
@@ -5,6 +5,25 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(...nums) {
+    if (nums.length === 0) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      for (const num of nums) {
+        sum += num;
+      }
+
+      return adder;
+    }
+  }
+
+  return adder;
```

Comment:
The `makeInfinityAdder` function is not handling the case where `adder` is called with a non-numeric argument. You should add a check to ensure that the argument is a number before adding it to the sum.